meinberlin/templates/a4dashboard/includes/nav_modules_item.html

```diff
@@ -23,6 +23,7 @@
         <div class="dashboard-nav__menu-content">
             <ul>
             {% for item in module_menu.menu %}
+                {% if not item.for_superuser or request.user.is_superuser %}
```

Comment:
@Rineee Concerning https://github.com/liqd/adhocracy4/pull/1216#pullrequestreview-1113438639
I check in the template. :grimacing:

Reply:
Haha, yes, I just found this PR, after my review there :)